content/projects/_index.md

```diff
@@ -23,7 +23,7 @@ order_by: "default"
     linkBgColor="#523971"
     gradient=""
     fontColor="#FFF">}}
-
+	<body class="projects">
```

Comment:
these shouldn't be body tags, the body tag is already open in the html document by this point.

Try `<div>` and make sure you close it further down with `</div>`

same feedback for the other changes here